src/ncp/ncp_base.cpp

```diff
@@ -55,6 +55,10 @@
 #include "openthread/jam_detection.h"
 #endif
 
+#if OPENTHREAD_ENABLE_RAW_LINK_API
+#include "openthread/link_raw.h"
+#endif
```

Comment:
Removed. And as @abtink's comment, we decide to not limit this new API to raw-link only.